### ISSUE TITLE
Anamorphic scaling

### ISSUE THREAD

**Issue by cjuniorfox** [hyprwm/Hyprland#4608]
### Description

In some cases, worth setting up anamorphic scaling, like when you're using an NTSC like CRT and setting up output models to deal with. The pixel clock for the horizontal resolution needs to be at least double the original horizontal resolution to have the proper timing for the blanking interval. So, you could have unusual output modes like
1440x240. To obtain the 720x480 output from that mode, you need to scale the horizontal resolution in half while scaling double the vertical resolution.
On X11, the scaling works well. You can play around on X11 using the Python script [chvideo.py](https://github.com/cjuniorfox/retropie-crt/blob/main/resources/usr/bin/chvideo.py) that I wrote originally for the Raspberry Pi but works ok on X11 if you add the '-m' flag.

**Comment by vaxerski** [comment]
what?

**Comment by vaxerski** [comment]
you can do that when mirroring, technically, as mirroring will stretch the image to fit.

**Comment by cjuniorfox** [comment]
Makes sense. The framebuffer will be created with specs for the main screen, while the second display will mirror the content from the same framebuffer.
Is my assumption correct?

**Comment by vaxerski** [comment]
(not exactly but the analogy works 1:1 so yes) and yes
